tests/regression/tests/REQUEST-934-APPLICATION-ATTACK-GENERIC/934110.yaml

```diff
@@ -0,0 +1,42 @@
+---
+  meta:
+    author: "Andrea Menin (theMiddle)"
+    description: "XML eXternal Entity"
+    enabled: true
+    name: 934110.yaml
+  tests:
+  -
+    test_title: 934110-1
+    desc: "Test XXE in file upload"
+    stages:
+    - stage:
+        input:
+          dest_addr: "127.0.0.1"
+          headers:
+            Host: "localhost"
+            User-Agent: "ModSecurity CRS 3 Tests"
+            Content-Type: "multipart/form-data; boundary=----WebKitFormBoundarysbZ9sZS2ONRQAD7q"
+          port: 80
+          method: POST
+          data:
```

Comment:
@theMiddleBlue Can you change this to be YAML compatible? That way it works with go-ftw also. The move should be:
* add `data: |`
* indent all lines below
* remove escaping on quotes
* remove starting `-`

Reply:
@theMiddleBlue ☝️ Can you handle this?